src/colorspaces.jl

```diff
@@ -1,19 +1,11 @@
 """
 Convert from sRGB to linear color space.
 """
-srgb2linear(u::Gray)::Gray = u ≤ 0.04045 ? 25 * u / 323 : ((200 * u + 11) / 211)^(12 / 5)
-srgb2linear(img::AbstractMatrix{<:Gray}) = srgb2linear.(img)
-function srgb2linear!(img::AbstractMatrix{<:Gray})
-    img .= srgb2linear.(img)
-    return nothing
-end
```

Comment:
By the way, usually in-place operations return the data instead of `nothing`. For example:

```julia
julia> x = [1, 3, 2]
3-element Vector{Int64}:
 1
 3
 2

julia> sort!(x)
3-element Vector{Int64}:
 1
 2
 3

julia> x
3-element Vector{Int64}:
 1
 2
 3
```